src/core/Callout/index.tsx

```diff
@@ -59,29 +66,66 @@ const Callout = ({
     }
   };
 
+  const getAction = () => {
+    if (expandable) {
+      return (
+        <ButtonIcon
+          onClick={() => {
+            setStage(expanded ? SDS_STAGE_CLOSED : SDS_STAGE_OPEN);
+          }}
+          sdsSize="small"
+          sdsType="tertiary"
+        >
+          <Icon
+            sdsIcon={expanded ? "chevronUp" : "chevronDown"}
+            sdsSize="s"
+            sdsType="button"
+          />
+        </ButtonIcon>
+      );
+    }
+    return onClose ? (
+      <ButtonIcon
+        onClick={handleClose}
+        sdsSize="small"
+        sdsType="tertiary"
+        size="large"
+      >
+        {" "}
+        <Icon sdsIcon="xMark" sdsSize="s" sdsType="iconButton" />{" "}
+      </ButtonIcon>
+    ) : null;
+  };
+
+  const expanded = !expandable || stage === "open";
+
+  let calloutTitle = children;
+  let calloutContent;
+
+  if (Array.isArray(children)) {
+    if (children[0].type.name === "CalloutTitle") {
+      [calloutTitle, ...calloutContent] = children;
+    } else {
+      // no CalloutTitle component, so use all child nodes as title
+      calloutTitle = children;
+    }
+  } else {
```

Comment:
both else statements results in the same action. wondering if we can combine the conditionals into a single if-statement (could stored each conditional in a named var first to make it readable if you want)?

Reply:
yea let me fix this up a bit.  I think I actually don't need either of the `calloutTitle = children`, because that's already assigned above.  I can just move the explanatory comments up to that variable initialization.